PSSlack/PSSlack.psd1

```diff
@@ -58,7 +58,7 @@ PowerShellVersion = '3.0'
 # NestedModules = @()
 
 # Functions to export from this module
-FunctionsToExport = '*'
+FunctionsToExport = @('Find-SlackMessage','Get-PSSlackConfig','Get-SlackAuth','Get-SlackChannel','Get-SlackGroup','Get-SlackGroupHistory','Get-SlackHistory','Get-SlackTeam','Get-SlackUser','New-SlackField','New-SlackMessage','New-SlackMessageAttachment','Send-SlackApi','Send-SlackFile','Send-SlackMessage','Set-PSSlackConfig','Test-SlackApi','Test-SlackAuth')
```

Comment:
Hiyo!  Sorry, been a little tied up!  Will try to merge this in tomorrow, thanks for adding it!

Would you mind switching this back to '*'?  We [swap it out in the build](https://github.com/RamblingCookieMonster/PSSlack/blob/master/psake.ps1#L63)

Thanks!